PowerPointLabs/PowerPointLabs/ActionFramework/PasteLab/PasteLabActionHandler.cs

```diff
@@ -39,7 +39,7 @@ protected sealed override void ExecuteAction(string ribbonId)
                 // Therefore we must capture the selection first.
                 ShapeRange selectedShapes = selection.ShapeRange;
 
-                // Save clipboard onto a temp slide, because CorruptionCorrrection uses Copy-Paste
+                // Save clipboard onto a temp slide
```

Comment:
This chunk seems redundant now, is it possible to just use IsClipboardEmpty to check instead?

Reply:
I realised that actually IsClipboardEmpty is being used on line 27 of the same file. Since IsClipboardEmpty is already used, this does seem redundant now.